cmd/utils/flags.go

```diff
@@ -2014,10 +2014,13 @@ func RegisterFilterAPI(stack *node.Node, backend ethapi.Backend, ethcfg *ethconf
 	filterSystem := filters.NewFilterSystem(backend, filters.Config{
 		LogCacheSize: ethcfg.FilterLogCacheSize,
 	})
-	stack.RegisterAPIs([]rpc.API{{
-		Namespace: "eth",
-		Service:   filters.NewFilterAPI(filterSystem),
-	}})
+
+	if !stack.AuctioneerEnabled() {
+		stack.RegisterAPIs([]rpc.API{{
+			Namespace: "eth",
+			Service:   filters.NewFilterAPI(filterSystem),
+		}})
```

Comment:
The filter apis involve `eth_newPendingTransactions` stream which shouldn't be enabled for auctioneer.